README.md

```diff
@@ -54,11 +54,14 @@ Install the dialonce-cordova plugin by executing:
 
 	$ cordova plugin add cordova-plugin-dialonce --variable ANDROID_API_KEY="<ANDROID_API_KEY>"
 
-
 ## Example
 
 	# Create initial Cordova app
 	$ cordova create myApp
 	$ cd myApp/
 	$ cordova platform add android
 	$ cordova plugin add cordova-plugin-dialonce --variable ANDROID_API_KEY="<ANDROID_API_KEY>"
+
+## API
+
+DialOnce SDK mostly work as autonomous component, one thing which need to be done is request for permissions: `navigator.dialonce.requestPermissions()`
```

Comment:
a little tweak to the text and also if you can describe why it is important it would be good.

`DialOnce SDK works as  and autonomous component. One thing which needs to be done is to request for permissions.`